### PR TITLE
Fix questions

### PR DESCRIPTION
**Pull request by ragusa87**
Fix for https://github.com/liip/RMT/issues/153

### CODE REVIEW

File: src/Liip/RMT/Prerequisite/TestsCheck.php [hunk 1]
```diff
@@ -25,7 +25,7 @@ class TestsCheck extends BaseAction
     public function __construct($options)
     {
         $this->options = array_merge(array(
-            'command' => 'phpunit --stop-on-failure',
+            'command' => './bin/phpunit --stop-on-failure',
```
Comment (jeanmonod) [inline src/Liip/RMT/Prerequisite/TestsCheck.php]: @ragusa87 Can we be sure that we will found a local bin script? According to https://phpunit.de/manual/6.5/en/textui.html the `phpunit` script must be in the path

Comment (ragusa87) [reply src/Liip/RMT/Prerequisite/TestsCheck.php]: @jeanmonod  You are right, I think it's best if you keep it as it was and drop the commit as I am not sure of what I did.

Furthermore `./bin/phpunit` is not standard. It should have been something like that: `./vendor/bin/phpunit`

Comment (jeanmonod) [reply src/Liip/RMT/Prerequisite/TestsCheck.php]: Can you fix your PR then?

Comment (ragusa87) [reply src/Liip/RMT/Prerequisite/TestsCheck.php]: done ;)

Comment (ragusa87) [reply src/Liip/RMT/Prerequisite/TestsCheck.php]: However, testing is still required...

File: src/Liip/RMT/Output/Output.php [hunk 2]
```diff
@@ -117,6 +117,11 @@ public function writeEmptyLine($repeat = 1)
     // QuestionHelper does about the same as we do here.
```
Comment (jeanmonod) [inline src/Liip/RMT/Output/Output.php]: @ragusa87 Have you seen that comment? I guess you are doing something similar...

**Comment by ragusa87** [comment]
TODO @ragusa87  Write tests

**Comment by ragusa87** [comment]
Tests added. Ready for review @Jonathanm10

File: src/Liip/RMT/Output/Output.php [hunk 3]
```diff
@@ -117,6 +117,11 @@ public function writeEmptyLine($repeat = 1)
...
     public function askQuestion(InteractiveQuestion $question, $position = null, InputInterface $input = null)
     {
+        if (class_exists('Symfony\Component\Console\Helper\QuestionHelper')) {
+            $helper = new \Symfony\Component\Console\Helper\QuestionHelper();
+            return $helper->ask($input, $this, $question->asSymfonyQuestion());
+        }
```
Comment (jeanmonod) [inline src/Liip/RMT/Output/Output.php]: Aren't you duplicate the code of line 127 here?  Why keeping the block starting at line 127?

**Pull request closed without merging**